docs/src/content/docs/commands/GETDEL.md

```diff
@@ -1,93 +1,90 @@
 
 ## Syntax
 
-```plaintext
+```
 GETDEL key
 ```
 
 ## Parameters
 
-- `key`: The key whose value you want to retrieve and delete. This parameter is a string and must be a valid key in the DiceDB database.
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `key`     | The key whose value you want to retrieve and delete.                      | String  | Yes      |
 
-## Return Value
+## Return values
 
-- `String`: If the key exists, the command returns the value associated with the key.
-- `nil`: If the key does not exist, the command returns `nil`.
+| Condition            | Return Value                                                     |
+|----------------------|------------------------------------------------------------------|
+| Key exists           | `String`: The command returns the value associated with the key. |
+| Key does not exist   | `nil`: The command returns `nil`.                                |
 
 ## Behaviour
 
 When the `GETDEL` command is executed, the following steps occur:
+  1. The command checks if the specified key exists in the DiceDB database.
+  2. If the key exists, the value associated with the key is retrieved.
+  3. The key is then deleted from the database.
+  4. The retrieved value is returned to the client.
+  5. If the key does not exist, `nil` is returned, and no deletion occurs.
 
-1. The command checks if the specified key exists in the DiceDB database.
-2. If the key exists, the value associated with the key is retrieved.
-3. The key is then deleted from the database.
-4. The retrieved value is returned to the client.
-5. If the key does not exist, `nil` is returned, and no deletion occurs.
-
-## Error Handling
+## Errors
 
 The `GETDEL` command can raise errors in the following scenarios:
 
-1. `Wrong Type Error`: If the key exists but is not a string (e.g., it is a list, set, hash, etc.), a `WRONGTYPE` error will be raised.
-2. `Syntax Error`: If the command is called without the required parameter, a syntax error will be raised.
+1. `Wrong Type Error`:
 
-## Example Usage
+   - Error Message: `ERROR WRONGTYPE Operation against a key holding the wrong kind of value`
+   - Occurs if the key exists but is not a string (e.g., it is a list, set, hash, etc.).
 
-### Example 1: Key Exists
+2. `Syntax Error`:
 
-```plaintext
-SET mykey "Hello, World!"
-GETDEL mykey
-```
+   - Error Message: `ERROR wrong number of arguments for 'getdel' command`
+   - Occurs if the command is called without the required parameter.
 
-`Output:`
+## Examples
 
-```plaintext
+### Example with Existent key
+
+```bash
+127.0.0.1:7379> SET mykey "Hello, World!"
+OK
+127.0.0.1:7379> GETDEL mykey
```

Comment:
Could you please add a get command after getdel to showcase that mykey is deleted from DiceDB.

Reply:
Oh, how did it get missed. My Bad let me add it immediately. And thanks for reverting back @apoorvyadav1111